pkg/crc/api/api_test.go

```diff
@@ -74,6 +75,40 @@ func TestApi(t *testing.T) {
 				"Success":          false,
 			},
 		},
+		{
+			command: "start",
+			args:    json.RawMessage(`{"pull-secret-file":"/Users/fake/pull-secret"}`),
+			expected: map[string]interface{}{
+				"Name":           "crc",
+				"Status":         "",
+				"Error":          "Incorrect argument name. Should be: pullSecretFile",
+				"KubeletStarted": false,
+				"ClusterConfig": map[string]interface{}{
+					"KubeConfig":    "",
+					"KubeAdminPass": "",
+					"ClusterAPI":    "",
+					"WebConsoleURL": "",
+					"ProxyConfig":   nil,
+				},
+			},
+		},
+		{
+			command: "start",
```

Comment:
If you implement this https://github.com/code-ready/crc/blob/master/pkg/crc/machine/fakemachine/client.go#L86 the test will be better.

Reply:
Done!